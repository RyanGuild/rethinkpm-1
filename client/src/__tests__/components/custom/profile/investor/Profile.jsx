import React from 'react';
import render from '../../../../../__utils__/renderHelper';
import Profile from '../../../../../components/custom/profile/investor/Profile';

test('render and snapshot', () => {
  let {container} = render(() => (
    <div>
      <Profile />
    </div>
  ));
  let inputs = container.querySelectorAll('a');
  expect(inputs.length).toEqual(3);
  expect(container).toMatchSnapshot();
});
