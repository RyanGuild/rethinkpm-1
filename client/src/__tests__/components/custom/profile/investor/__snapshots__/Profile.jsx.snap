// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render and snapshot 1`] = `
<div>
  <div>
    <h3
      class="text-primary"
    >
      Investor Profile
    </h3>
    <p
      class="lead"
    >
      <i
        class="fas fa-user"
      />
       Welcome
    </p>
    <div
      class="dash-buttons"
    >
      <a
        class="btn btn-light"
        href="/edit-profile"
      >
        <i
          class="fas fa-user-circle text-primary"
        />
         Edit Profile
      </a>
      <a
        class="btn btn-light"
        href="/add-experience"
      >
        <i
          class="fab fa-black-tie text-primary"
        />
         Add Experience
      </a>
      <a
        class="btn btn-light"
        href="/add-education"
      >
        <i
          class="fas fa-graduation-cap text-primary"
        />
         Add Education
      </a>
    </div>
    <div>
      <h2
        class="my-2"
      >
        My Properties
      </h2>
      <table
        class="table"
      >
        <thead>
          <tr>
            <th>
              Company
            </th>
            <th
              class="hide-sm"
            >
              Title
            </th>
            <th
              class="hide-sm"
            >
              Years
            </th>
            <th />
          </tr>
        </thead>
        <tbody />
      </table>
      <p>
        ooh ya
      </p>
    </div>
  </div>
</div>
`;
